client/src/hooks/use-mobile.test.tsx

```
@@ -0,0 +1,198 @@
+      dispatchEvent: vi.fn(),
+    };
+    window.matchMedia = vi.fn().mockImplementation(() => matchMediaMock);
+  });
+
+  afterEach(() => {
+    // Restore original window properties
+    if (originalInnerWidth) {
+      Object.defineProperty(window, 'innerWidth', originalInnerWidth);
+    } else {
+      // If it wasn't originally defined, delete it
+      delete (window as any).innerWidth;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)